src/js/elements/subindicator_filter/filter_controller.js

```diff
@@ -335,8 +333,13 @@ export class FilterController extends Component {
     addPreviouslySelectedFilters(group, isDefault, isPreviouslySelected) {
         let filterRow = this.addEmptyFilter(isDefault, false, false, false, isPreviouslySelected);
 
-        filterRow.setPrimaryIndexUsingValue(group.group);
-        filterRow.setSecondaryIndexUsingValue(group.value);
+        try {
+            filterRow.setPrimaryIndexUsingValue(group.group);
+            filterRow.setSecondaryIndexUsingValue(group.value);
```

Comment:
What caused this to break, it would be better to know why it broke and fix the cause
I think this change risks hiding errors and failing silently

Reply:
@michaelglenister if an item is not existing, the dropdown throws an error. 

https://github.com/OpenUpSA/wazimap-ng-ui/blob/7bd605a454552f4cc90bfaa0562e35fa5b0f4c32/src/js/ui_components/dropdown.js#L41

but we don't want to break the UI because of this error so this error should fail silently. we have the same structure for default filters in `addDefaultFilter()` and for non -aggregatable in `addNonAggregatableFilter()`